internal/pkg/util/yamlprotoconverter.go

```diff
@@ -0,0 +1,19 @@
+package yamlprotoconverter
+
+import (
+	"github.com/ghodss/yaml"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+func FromYAMLToProto(yml string, pb proto.Message) error {
```

Comment:
I know this is pretty self explanatory, but can you document evert exported func? Not quite sure why golint isn't picking up on this.